plugins/upstash/api_key_test.go

```diff
@@ -0,0 +1,59 @@
+package upstash
+
+import (
+	"testing"
+
+	"github.com/1Password/shell-plugins/sdk"
+	"github.com/1Password/shell-plugins/sdk/plugintest"
+	"github.com/1Password/shell-plugins/sdk/schema/fieldname"
+)
+
+func TestAPIKeyProvisioner(t *testing.T) {
+	plugintest.TestProvisioner(t, APIKey().DefaultProvisioner, map[string]plugintest.ProvisionCase{
+		"default": {
+			ItemFields: map[sdk.FieldName]string{
+				fieldname.APIKey: "d68850db-69f7-qxe9pubcmjnqfgyexample",
+				fieldname.Email:  "fakememail12@gmail.com",
```

Comment:
I suggest we update this email address to wendy@appleseed.com, which is a standard email address we use throughout our branding. We seem to be using it on the Zendesk shell plugin too:

https://github.com/1Password/shell-plugins/blob/a39d64acfb1dd3cdd496507d54cd4c8306da6f78/plugins/zendesk/api_token_test.go#L16

Reply:
@arunsathiya I will do changes

Reply:
Marking as resolved in in https://github.com/1Password/shell-plugins/pull/316/commits/d43d07836385c9388a54d27f591c0cb714249432.